Library/Formula/seafile.rb

```diff
@@ -0,0 +1,56 @@
+class Seafile < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/seafile/archive/v4.1.3.tar.gz"
+  sha1 "4834f6399e47ab6426384f367e264fdb021e064b"
+
+  head "https://github.com/haiwen/seafile.git", :branch => "4.1"
+
+  #[FIX] fix openssl build
+  patch :p1 do
+    url "https://github.com/Chilledheart/seafile/commit/79fc942d.diff"
+    sha1 "a4c81dbf6e131502b2c229b9cab2f324e8c51e5d"
+  end
+
+  #[FIX] homebrew autotools path
+  patch :p1 do
+    url "https://github.com/Chilledheart/seafile/commit/5848eb75.diff"
+    sha1 "b8b0d17ae03474e996ce7e2a42aefe0edb80d159"
+  end
+
+  #[FIX] use system zlib
+  patch :p1 do
+    url "https://github.com/Chilledheart/seafile/commit/239c148b.diff"
+    sha1 "da429338d1726b95e26b70f7a9ce1bfa3a7392ae"
+  end
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!